app/build.gradle.kts

```
@@ -28,6 +28,8 @@ android {
         multiDexEnabled = true
 
         resourceConfigurations += listOf("ar", "bg", "bn", "bn-rIN", "bs", "cs", "da", "de", "el-rGR", "en", "eo", "es", "es-rAR", "fi", "fr", "he-rIL", "hi", "hr", "hu", "in-rID", "is", "it", "ja", "ko", "lt", "lv", "nb-rNO", "nl", "oc", "pl", "pt-rPT", "ro-rRO", "ru", "sk", "sl", "sv", "tr", "uk", "vi", "zh-rCN", "zh-rTW")
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunnerArguments(mapOf("clearPackageData" to "true"))
```

This version seems to be deprecated:

```
'testInstrumentationRunnerArguments(Map<String, String>): Unit' is deprecated. Replaced by testInstrumentationRunnerArguments property
```

The suggestion I gave you earlier, which I took directly from Google's documentation, is probably the right one (see https://github.com/CatimaLoyalty/Android/pull/1900/files#r1614554689 and https://developer.android.com/training/testing/instrumented-tests/androidx-test-libraries/runner):

```suggestion
        testInstrumentationRunnerArguments clearPackageData: 'true'
```

---

I tried the syntax that you proposed and was also mentioned in the docs but some errors occurred. I think it is because the proposed syntax is in Groovy DSL, while it should be in Kotlin DSL, so I adjust it accordingly.